.flake8

```diff
@@ -1,29 +1,38 @@
+# TODO: move this to pyproject.toml when supported, see https://github.com/PyCQA/flake8/issues/234
+
+
 [flake8]
+indent-size = 4
+max-complexity = 10
```

Comment:
What does this do?

Reply:
Those are arguments for the linter. (WIP)